Spectrum_Peak_Visualization.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import peakutils
+import SPEFile
+
+"""
+plot_spectrum plots the spectra only.
+"""
+def plot_spectrum(spectrum, title,energy_range='None'):
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    counts = spectrum.data
+    
+    #Graphs the spectrum first without any ROIs.
+    plt.semilogy(energy_axis,counts,drawstyle='steps-mid')
+    plt.xlabel('Energy (keV)')
+    plt.ylabel('Counts')
+    plt.title(title)
+    
+    """
+    If energy_range is set to a tuple of two values, then below code will graph
+    the spectrum within the energy_range values for the energies (x-values).
+    """    
```

Comment:
Don't use strings as comments. See [this post](http://stackoverflow.com/questions/708649/python-comments-vs-strings) for explanation.
As the accepted answer describes, docstrings explain _what_ the code does, while comments describe _how_ the code is doing it.